applications/luci-app-acme/htdocs/luci-static/resources/view/acme.js

```diff
@@ -86,10 +86,12 @@ return view.extend({
 
 		o = s.taboption('challenge_dns', form.ListValue, 'dns', _('DNS API'),
 			_("To use DNS mode to issue certificates, set this to the name of a DNS API supported by acme.sh. " +
-				"See https://github.com/acmesh-official/acme.sh/wiki/dnsapi for the list of available APIs. " +
+				'See %s for the list of available APIs. ' +
 				"In DNS mode, the domain name does not have to resolve to the router IP. " +
 				"DNS mode is also the only mode that supports wildcard certificates. " +
-				"Using this mode requires the acme-dnsapi package to be installed."));
+				"Using this mode requires the acme-dnsapi package to be installed.")
+				.format('<a href="https://github.com/acmesh-official/acme.sh/wiki/dnsapi" target="_blank">DNS API</a>')
```

Comment:
This means the link target won't be translated, right? That's a bit odd...

Reply:
That's a bold assumption :)
Also, further down you're doing the same with "DNS alias mode" which definitely sounds like something that should be translatable...

Reply:
Acronyms and abbreviations such as DNS and API almost never get translated any more because they are so ubiquitous - a quick look [through weblate](https://hosted.weblate.org/search/openwrt/?q=dns&sort_by=-priority%2Cposition&checksum=) should give you a good idea.

Reply:
It's true that they don't need to be translated, however the combination of them may differ for different languages; for example in Swedish (native speaker) it would be correct to write `DNS-API` (or `DNS-värdtjänst` if you translate the API part), with a hyphen between them. So making that assumption is at least definitely not correct, however if we need to care about it that's a different question.

Reply:
I generally agree that most stuff should be i18n able, but ossified abbreviations can be skipped.